content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -0,0 +1,465 @@
+
+### Properties:
+  * `helm_config` - Represents Helm configuration.
+    
+    *type:* cloudify.types.helm.HelmConfig
+    
+    Currently contains `executable_path` with default value of '/usr/bin/helm'.
+  * `use_existing_resource` - If true, use an existing helm installation rather than installing it.
+    
+    *type:* boolean
+    
+    *default:* False
+  * `installation_source` - Location to download the Helm installation from. Ignored if `use_existing_resource` is true.
+    
+    *type:* string
+    
+    *default:* ''
+    
+    You can see helm releases [here](https://github.com/helm/helm/releases) please use helm 3.X.X version.
+
+Helm plugin uses `curl` on  `installation_source` and unzip it, then move it to `executable_path` or to default location if `executable_path` is not provided.
+
+### Example:
+
+{{< highlight  yaml  >}}
+
+inputs:
+
+  helm_executable:
+    description: >
+      Helm binary path.
+    type: string
+    default: '/tmp/helm'
+
+  helm_installation_source:
+    description: >
+      Helm download link.
+    type: string
+    default: 'https://get.helm.sh/helm-v3.3.1-linux-amd64.tar.gz'
+
+dsl_definitions:
+
+  helm_config: &helm_config
+    executable_path: { get_input: helm_executable }
+
+node_templates:
+
+  helm_install:
+    type: cloudify.nodes.helm.Binary
+    properties:
+      helm_config: *helm_config
+      use_existing_resource: false
+      installation_source: { get_input: helm_installation_source }
+
+{{< /highlight >}}
+
+## cloudify.nodes.helm.Repo
+
+This node type responsible for adding repositories to Helm client using `helm repo add` command.
+
+### Properties
+  * `helm_config` - Represents helm configuration.
+  
+    *type:* cloudify.types.helm.HelmConfig
+    
+    Currently contains `executable_path` with default value of '/usr/bin/helm'.
+  * `use_external_resource` - Indicate whether the resource exists or if Cloudify should create the resource,
+    true if you are bringing an existing resource, false if you want cloudify to create it.
+    In this case it means cloudify will use a repo that already exists on helm client.
+    
+    *type*: boolean
+    
+    *default*: false
+  * `resource_config` - dictionary represents repo configuration.
+    
+    Contains:
+    
+     * `name` - Name of the repo that added/removed.
+     
+        *type*: string
+     
+        *required*: true
+     * `repo_url` - URL of the repo to add.
+     
+        *type*: string
+     
+        *required*: true
+     * `flags` -  list of flags add to both "helm repo add" and "helm repo remove" commands. For example:
+{{< highlight  yaml  >}}      
+- name: namespace
+  value: my_namespace
+{{< /highlight >}}
+
+      If the flag not requires value, omit "value" and specify only the name as element in the list.
+        *default*: []
+        
+**Notes**:
+
+* Currently, Helm plugin support add repo only via url.
```

Comment:
Double check with @natishalom if he likes this limitation.

Reply:
Well its my fault because generally helm `repo add` command supports URL. 
https://helm.sh/docs/helm/helm_repo_add/ 

removing this note.